uint8 system_absolut = 1
uint8 system_home = 2
uint8 system_base = 3
uint8 system_offset_from_target  = 4
uint8 system_offset_from_here  = 5

uint8 course_abs=0
uint8 course_from_move_dir=1
uint8 course_add=2

uint8 mode_allow_break_distance = 0
uint8 mode_not_break_distance = 1

uint8 fly_simple=1
uint8 fly_safe=2
uint8 landing_simple=3
uint8 landing_aruco=4

uint8 fly_mode

uint8 system
uint8 break_distance_mode

float64 data_x
float64 data_y
float64 data_z

float64 speed

uint8 course_mode
float64 course
float64 acc
